validation/gwas/method/regenie/tasks.py

```diff
@@ -0,0 +1,109 @@
+import glob
+import os
+import shutil
+from pathlib import Path
+
+import yaml
+from invoke import task
+
+HAILPY = os.environ.get("HAIL_PYTHON_EXECUTABLE", "/opt/conda/envs/hail/bin/python")
+GLOWPY = os.environ.get("GLOW_PYTHON_EXECUTABLE", "/opt/conda/envs/glow/bin/python")
+BASEPY = os.environ.get("BASE_PYTHON_EXECUTABLE", "/opt/conda/bin/python")
+DEFAULT_TEST_DATADIR = os.getenv("TEST_DATADIR", "../../../../sgkit/tests/test_regenie")
```

Comment:
For relative paths it's probably better to start from `__file__` to avoid depending on CWD.

Reply:
https://github.com/pystatgen/sgkit/pull/66/commits/64c1da651f5cd2596f28461639981809f80519ad#diff-7240a770aa7f50f89b47ef6286c19a81R17